apps/test/unit/lib/script-editor/StageDescriptionsTest.js

```diff
@@ -129,6 +129,7 @@ describe('StageDescriptions', () => {
         ]
       })
     );
+    wrapper.update();
```

Comment:
Necessary because the fake server response above no longer implicitly triggers an update by the enzyme wrapper.